code/game/machinery/doors/door.dm

```diff
@@ -109,6 +109,10 @@ var/list/all_doors = list()
 
 	if(allowed(user))
 		open()
+	else if(isanimal(user))
+		var/mob/living/simple_animal/SA = user
+		if(SA.can_force_doors)
+			open()
```

Comment:
May want to change this up, as right now it means the big thing with scary claws can open a door instantly, but also can be locked down via door bolts.

Maybe rather than having can_force_doors act as a boolean, have it variable, with 1 being can open doors after some time but not bolted doors, 2 being can open doors that are bolted/welded/jammed, but it takes a bit of time